api/oci/extensions/repositories/ocireg/type.go

```diff
@@ -128,6 +130,18 @@ func (a *RepositorySpec) Repository(ctx cpi.Context, creds credentials.Credentia
 	return NewRepository(ctx, a, info)
 }
 
+func (a *RepositorySpec) Validate(ctx cpi.Context, creds credentials.Credentials, context ...credentials.UsageContext) error {
+	info, err := a.getInfo(creds)
+	if err != nil {
+		return err
+	}
+	h, p, _ := info.HostInfo()
+	if p == "" {
+		p = "443"
+	}
```

Comment:
should we log here something like:
'No port provided falling back to default port 443'

Reply:
this is the default port used for OCI registry REST access. I think a log is not required, here